test_bindings.py

```diff
@@ -35,6 +38,38 @@ def monitor_task(task_href):
         print("The task did not finish successfully.")
         exit()
 
+def upload_file_in_chunks(file_path):
+    """Uploads a file using the Uploads API
+
+    The file located at 'file_path' is uploaded in chunks of 200kb.
+
+    Args:
+        file_path (str): path to the file being uploaded to Pulp
+
+    Returns:
+        upload object
+    """
+    with open(file_path, 'rb') as full_file:
+        size = os.stat(file_path).st_size
+        chunk_size = 200000
+        offset = 0
+        md5hasher = hashlib.new('md5')
+        while True:
+            chunk = full_file.read(chunk_size)
+            if not chunk:  # if chunk == b'':
```

Comment:
is the comment here just stale code?

Reply:
It's copy/paste from pulp-smah utils that are used to do uploads in pulp 2 functional tests. I can remove it because it's pretty obvious how the end of file is determined.